.github/ISSUE_TEMPLATE/BUG-ISSUE.yml

```
@@ -2,68 +2,86 @@ name: Bug Report
 description: Create a report to help us improve
 labels: ["Type: Bug"]
 body:
+  - type: markdown
+    attributes:
+      value: |
+        Note: Internal developers should consider opening a private issue if there are sensitive details that would be helpful to include
+        Links to private resources (ie logs, slack conversations, etc.) can be shared in public issues
```

This is a nit, but maybe invert the order (and broaden scope so it could be field engineers or other Soloists) so that they don't read the first sentence and just immediately go to the private repo?
```suggestion
        **Note for internal Solo folks**: You can share links to private resources that require login in public issues, like logs, Slack conversations, etc. If you have more sensitive details that would be helpful to include, consider opening a [private issue instead](https://github.com/solo-io/solo-projects/issues/new/choose).
```